### DISCUSSION TITLE
IFUSY camera after installing firmware 0.3.0

### DISCUSSION THREAD

**Discussion by ostap7474** [roleoroleo/yi-hack-Allwinner-v2#747] [Q&A]
The IFUSY camera works after installing firmware 0.3.0, but if you turn off the power and turn it on again, the camera asks to connect Wi-Fi through the application. I connect Wi-Fi through the app and the camera works again until the power is turned off. After the power is turned off, history repeats itself. Restoring the camera from a backup did not help. I returned to firmware 0.2.6 and it works without problems. (Serial number IFUSY65BGYKCU8201207, Model suffix y211ga, Basic version 9.0.36.02_202105081647)

**Comment by roleoroleo** [comment]
There are too many differences between 0.2.6 and 0.3.0.
I can't find the problem.

**Comment by ostap7474** [comment]
I installed version 0.2.7 and it works great. Firmware 0.2.8 causes the same problems as 0.3.0

**Comment by roleoroleo** [comment]
I have checked all the changes between version 0.2.7 and 0.2.8 one by one
https://github.com/roleoroleo/yi-hack-Allwinner-v2/compare/0.2.7...0.2.8
and I can't understand what the problem could be.
